test/verify/check-terminal

```diff
@@ -193,6 +194,31 @@ PROMPT_COMMAND='printf "\\033]0;%s@%s:%s\\007" "${USER}" "${HOSTNAME%%.*}" "${PW
         b.wait_present('.terminal')
         b.wait_attr(".xterm-viewport", "style", white_style)
 
+    @nondestructive
+    def testOnlyTerminal(self):
+        b = self.browser
+        m = self.machine
+
+        m.write("/etc/cockpit/cockpit.conf", "[WebService]\nShell = /system/terminal.html\n")
+        m.start_cockpit()
+        b.open("/")
+
+        b.try_login("admin", "foobar")
+        b.expect_load()
+        b.wait_present("#terminal")
+
+        # wait until first line is not empty
+        n = 1
+        b.wait_present(".terminal .xterm-accessibility-tree")
+        b.wait_js_func("(function (sel) { return ph_text(sel).trim() != ' '})", line_sel(n))
+
+        # execute a command we can see the effect of
+        b.wait_js_cond("ph_text('.terminal').indexOf('uid=') == -1")
+        b.key_press("id\r")
+        b.wait_js_cond("ph_text('.terminal').indexOf('uid=') >= 0")
+
+        # Cleanup
+        m.execute("rm /etc/cockpit/cockpit.conf")
```

Comment:
Please drop this. It's not necessary as nonDestructiveSetup() cleans it up. And it's also not correct as a pattern, as it would leave the file behind if the test fails in the middle (and thus break every subsequent test).

Reply:
This was my fault :D sorry

Reply:
No, we only restart the machine on "test harness" errors such as ssh connections timing out.

Reply:
Roger. Removed.